app/models/concerns/database_indexable.rb

```diff
@@ -39,6 +39,7 @@ def generate_searchable_content # rubocop:disable Metrics/AbcSize,Metrics/Method
         organisations.map(&:local_authority_within).reject(&:blank?).uniq,
         organisations.map(&:county).reject(&:blank?).uniq,
         organisations.map(&:region).reject(&:blank?).uniq,
+        VacancyPresenter.new(self).readable_visa_sponsorship_availability,
```

Comment:
There was an interesting bug on the review app where when a user entered "visa" as a keyword, the visa_sponsorship_available filter was selected but 0 vacancies were found. When the visa_sponsorship_available was ticket without any keyword, 2 vacancies were found.

This was due to the full text search not finding any mention of the words "visa" in these jobs, despite the fact that the field visa_sponsorship_available was set to true.

In order to fix this behaviour, I added the readable_visa_sponsorship_availability method to the vacancy presenter class and added this to the searchable_content so that any vacancies with visa_sponsorship_available set to true would have "visa sponsorship" added to the searchable_content field.